app/retail/templates/emails/kudos_mkt.html

```diff
@@ -80,8 +80,8 @@ <h2>{% trans "Kudos can be used to" %}</h2>
 <p  style="line-height: 2; font-size:18px;">
   {% blocktrans %}
   Recognize Gitcoiners you’ve worked with <br>
-  Understand other Gitcoiner skillsets<br>
-  See who Gitcoiners have worked iwth<br>
+  Understand other Gitcoiner's skillsets<br>
```

Comment:
Should this be `Gitcoiners'`? I'm not a grammar expert though....

Reply:
sorry, finally got to this. yes you are correct, will push this small change.